src/main.ts

```diff
@@ -424,25 +439,23 @@ export default class RecipeGrabber extends Plugin {
 			if (!type) {
 				return false;
 			}
-			let path;
+			let path = "";
 			if (this.settings.imgFolder == "") {
-				//@ts-ignore
```

Comment:
eslint didn't want any use of `ts-ignore`. but thinking the `any` fix here should do the trick.